src/driver.cpp

```diff
@@ -7,8 +7,12 @@
 
 struct yy_buffer_state;
 
-extern struct yy_buffer_state *yy_scan_string(const char *yy_str,
-                                              yyscan_t yyscanner);
+#ifndef MIN_YY_BUF_SIZE
```

Comment:
This is not some preprocessor definition exported by flex, right? IMO better to drop the `#ifndef` - it's confusing b/c it has no users (overrides) in our codebase.